app/api/data.py

```diff
@@ -1,6 +1,7 @@
 """Registers the necessary routes for the core data model. """
+import flask
 
-from flask import jsonify, request, current_app, abort
+from flask import current_app, jsonify
```

Comment:
since we're importing flask in its entirety, let's just use `flask.current_app` and `flask.jsonify` also

Reply:
Done.